drivers/network/afd/afd/write.c

```diff
@@ -645,22 +647,55 @@ AfdPacketSocketWriteData(PDEVICE_OBJECT DeviceObject, PIRP Irp,
         Status = QueueUserModeIrp(FCB, Irp, FUNCTION_SEND);
         if (Status == STATUS_PENDING)
         {
-            Status = TdiSendDatagram(&FCB->SendIrp.InFlightRequest,
-                                     FCB->AddressFile.Object,
-                                     SendReq->BufferArray[0].buf,
-                                     SendReq->BufferArray[0].len,
-                                     TargetAddress,
-                                     PacketSocketSendComplete,
-                                     FCB);
-            if (Status != STATUS_PENDING)
-            {
-                NT_VERIFY(RemoveHeadList(&FCB->PendingIrpList[FUNCTION_SEND]) == &Irp->Tail.Overlay.ListEntry);
-                Irp->IoStatus.Status = Status;
-                Irp->IoStatus.Information = 0;
-                (void)IoSetCancelRoutine(Irp, NULL);
-                UnlockBuffers(SendReq->BufferArray, SendReq->BufferCount, FALSE);
-                UnlockRequest(Irp, IoGetCurrentIrpStackLocation(Irp));
-                IoCompleteRequest(Irp, IO_NETWORK_INCREMENT);
+            if (SendReq->BufferCount > 1) {
```

Comment:
Please keep the code style of the rest of the file.

Reply:
The code style of this file was already very mixed and most did not match the coding style guide. I went through the file and fixed the control structures throughout the file to match the coding style. I had originally followed the content style of the file rather than the guide.. hopefully this is what you intended.